### DISCUSSION TITLE
ghost-vs-docusaurus

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#26] [Blog comments]
# ghost-vs-docusaurus

Почти закончил переезд с Ghost на Docusaurus, что дает целый ряд преимуществ.

https://blog.amd-nick.me/ghost-vs-docusaurus

**Comment by CMAULTOP** [comment]
wiki.nikiv.dev уже не docusaurus, своё сделали

**Reply by AMD-NICK** [reply]
Да, я видел, что он ввел платную подписку даже, но движок вроде все еще Docusaurus

**Reply by CMAULTOP** [reply]
нет, точно что-то другое, но из этой темы
![image](https://github.com/AMD-NICK/blog.amd-nick.me/assets/55154964/2be54403-2963-4836-8132-0bbb07227136)

**Reply by AMD-NICK** [reply]
VitePress. Интересненько
